src/collection/views/CollectionDetail.tsx

```diff
@@ -865,32 +869,54 @@ const CollectionDetail: FunctionComponent<CollectionDetailProps> = ({
 												<span className="u-d-block">-</span>
 											)}
 										</p>
+										<p className="u-text-bold">
+											<Trans i18nKey="collection/views/collection-detail___vakken">
+												Vakken
+											</Trans>
+										</p>
+										<p className="c-body-1">
+											{lom_classification && lom_classification.length ? (
+												generateSearchLinks(
+													id,
+													'subject',
+													lom_classification
+												)
+											) : (
+												<span className="u-d-block">-</span>
+											)}
+										</p>
 									</Spacer>
 								</Column>
 								<Column size="3-3">
 									<Spacer margin="top">
+										<p className="u-text-bold">{t('Aangemaakt op')}</p>
+										<p className="c-body-1">{formatDate(created_at)}</p>
 										<p className="u-text-bold">
-											<Trans i18nKey="collection/views/collection-detail___laatst-aangepast">
-												Laatst aangepast
-											</Trans>
+											{t(
+												'collection/views/collection-detail___laatst-aangepast'
+											)}
 										</p>
 										<p className="c-body-1">{formatDate(updated_at)}</p>
 									</Spacer>
 								</Column>
-								<Column size="3-6">
-									<Spacer margin="top">
-										<p className="u-text-bold">
-											<Trans i18nKey="collection/views/collection-detail___ordering">
-												Ordering
-											</Trans>
-										</p>
-										{!!get(collection, 'relations', []).length && (
-											<p className="c-body-1">
-												<Trans i18nKey="collection/views/collection-detail___deze-collectie-is-een-kopie-van">
-													Deze collectie is een kopie van:
+								{(hasCopies || hasParentBundles) && (
+									<Column size="3-6">
+										<Spacer margin="top">
+											<p className="u-text-bold">
+												<Trans i18nKey="collection/views/collection-detail___ordering">
+													Ordering
 												</Trans>
-												{(get(collection, 'relations', []) as any[]).map(
-													(relation: any) => (
+											</p>
+											{hasCopies && (
+												<p className="c-body-1">
+													<Trans i18nKey="collection/views/collection-detail___deze-collectie-is-een-kopie-van">
+														Deze collectie is een kopie van:
+													</Trans>
+													{(get(
+														collection,
+														'relations',
+														[]
+													) as any[]).map((relation: any) => (
```

Comment:
Hier mogelijks een todo bij want lijkt me dat we hier wel ng een type voor kunnen voorzien?